src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SEPARATOR = " ";
+    public static final String DELIMITER = ",";
+    public static final int NAME = 0;
+    public static final int VALUE = 1;
+    public static final int SUPPLY = 0;
+    public static final int BUY = 1;
+    public static final int RESULT = 2;
+    public static final int LINES_IN_REPORT = 3;
+    public static final String[] NAME_VALUES = {"supply", "buy", "result"};
```

Comment:
1. any reason to make them public?
2. dont have wrong impression that you must create constants for every value
```suggestion
```